### DISCUSSION TITLE
Not found I define desactivate method in CodeIgniter\Shield\Models\UserModel

### DISCUSSION THREAD

**Discussion by oniricosistemas** [codeigniter4/shield#754] [Q&A]
### PHP Version

8.2.4

### CodeIgniter4 Version

4.3.3

### Shield Version

1.0.0-beta

### Which operating systems have you tested for this bug?

Windows

### Which server did you use?

apache

### Database

10.4.28-MariaDB

### Did you customize Shield?

no

### What happened?

Not found I define desactivate method in CodeIgniter\Shield\Models\UserModel

### Steps to Reproduce

$users = auth()->getProvider();
$user = $users->findById($id);
$users->desactivate($user);

### Expected Output

Call to undefined method CodeIgniter\Shield\Models\UserModel::desactivate

**Comment by kenjis** [comment]
If you want to deactivate a user, call `$user->deactivate()`.
See https://codeigniter4.github.io/shield/authorization/#activating-a-user

If you defined `UserModel::desactivate`, can you show the source code?
